ui/lobby/css/app/_app.scss

```diff
@@ -1,52 +1,34 @@
-@import 'pool';
-@import 'hook-chart';
+@import 'quick';
+@import 'hook-graph';
 @import 'hook-filters';
 @import 'hook-list';
 
 .lobby__app {
   @extend %flex-column;
 
   @include lobby-app-size;
+  background-size: auto 80%;
+  background-position: center top 70%;
+  background-repeat: no-repeat;
+  $logo-color: '242320';
```

Comment:
![image](https://github.com/user-attachments/assets/9e9fdd5f-7006-4826-950a-fb6e397138d2)

unless you have a very good screen the logo has become almost invisible there. Dark theme and quick pairing are the default render for the homepage.

Reply:
almost invisible is almost as good as invisible